README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
+   + here is the link to watch the best Trek while stuck inside. [You'll thank me](https://www.netflix.com/title/70158330?source=35) 
```

Comment:
Enjoyed the emphatic recommendation and as Dora said, creative way to add the link title. I will also say good job on adding the text to the link (You'll thank me"). I personally did not do this, didn't realize we had to or else a simple 'link' would show. As a result yours looks clean and brings together the spirit of what you were trying to achieve.